pkg/mcp/loader.go

```diff
@@ -148,6 +148,29 @@ func (sm *SessionManager) ShutdownServer(ctx context.Context, server ServerConfi
 	return nil
 }
 
+// RestartK8sDeployment restarts the Kubernetes deployment by deleting its pods, which will trigger a restart.
+func (sm *SessionManager) RestartK8sDeployment(ctx context.Context, server ServerConfig) error {
+	id := sessionID(server)
```

Comment:
```suggestion
	if server.Command == "" {
		return nil
	}
	id := sessionID(server)
```

Reply:
Updated. In which case server.command is empty and why should we avoid it?